### PR TITLE
Issue #15329: Updated coverage table of google style guide implementation for section 4.4 Column Limit

### PR DESCRIPTION
**Pull request by Zopsss**
#15329 

Updated coverage table for section 4.4 Column Limit: 100, added a message indicating how to use suppression for long identifiers.

### CODE REVIEW

File: src/xdocs/google_style.xml [hunk 1]
```diff
@@ -798,6 +799,11 @@
                   <a href="https://github.com/checkstyle/checkstyle/issues/14938">
                     #14938</a>.
                   <br />
+                  <br />
+                  To suppress "LineLength" violations for long identifiers,
+                  use following annotation:
+                  <code>@SuppressWarnings({"LineLength"})</code>
```
Comment (romani) [inline src/xdocs/google_style.xml:805]: Please postpone this update until we agree in that issue.

For now Please update table with comment: `long indentures will be handled at https://github.com/checkstyle/checkstyle/issues/15233`

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/google_style.xml (hunk 1 above)
Comment (Zopsss) [reply src/xdocs/google_style.xml]: done

**Review by romani — APPROVED** [review]
ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**